src/components/Nav/UserDropdown.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { Dropdown, Icon, MenuItem } from 'patternfly-react';
+// import * as UserAction from '../../actions/UserAction';
+
+type UserProps = {
+  dispatch?: any;
+};
+
+class UserDropdown extends React.Component<UserProps, {}> {
+  userLogout = () => {
+    // const { dispatch } = this.props;
+    sessionStorage.removeItem('user');
+    // dispatch(UserAction.logout());
+    window.location.reload();
```

Comment:
Invoking the window is always like access to bytecode.
Perhaps this can be done through the React Router ?

Reply:
I asked about this when I opened the PR but nobody answer me about How I can do it? Because I tried but I couldn't

Reply:
Should I create a static context and call to this.context.router.transitionTo(/path-to-link); ?

Reply:
I fixed...I don't know why I couldn't use it before